lib/util/memcache.ex

```diff
@@ -4,12 +4,51 @@ use Croma
 
 defmodule Antikythera.Memcache do
   @moduledoc """
-  In-memory cache for each executor pool.
+  Easy-to-use in-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  #{inspect(__MODULE__)} behaves as a key-value storage.
+  Cached key-value pairs are internally stored in ETS.
+  It accepts arbitrary (but not too large) terms as both keys and values.
+
+  ## Usage
+
+      iex> Antikythera.Memcache.write("foo", "bar", epool_id, 3_600)
       :ok
       iex> Antikythera.Memcache.read("foo", epool_id)
       {:ok, "bar"}
+
+  ## Limits
+
+  The number of records and the size of keys and values is limited.
+
+  - The maximum number of records for each executor pool is #{AntikytheraCore.ExecutorPool.MemcacheWriter.max_records()}.
+      - If exceeds the limit, records whose expiration time is nearest are evicted.
```

Comment:
This characteristic is wee bit tricky to convey properly, could use a little improvements to my eyes.

Let me clarify: Do you want to state (or imply) "multiple records can be evicted AT ONCE on a single write" here?
I think this is true (although rarely, perhaps on highly concurrent write?), looking at implementation of `evict_head_records_if_exceeds_limit/1`.

If so, how about: "If exceeds the limit, records nearest to expiration are evicted so that a new record can be inserted."
Suggestions welcomed.

Reply:
Usually, only one record is evicted if exceeds the limit because there is only one writer process (`AntikytheraCore.ExecutorPool.MemcacheWriter`).
However, changing the maximum record number with hot upgrades will cause eviction of multiple records (see this conversation https://github.com/access-company/antikythera/pull/14#discussion_r196703377).

So, eviction of multiple records can rarely occurs and it would be better to change "records" to "a record".

Reply:
Thanks for redirection, I forgot about https://github.com/access-company/antikythera/pull/14#discussion_r196703377
Yes, if so changing to "a record" should be ok.